src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -1,11 +1,133 @@
 import React from 'react';
+import Box from '../../../components/Box';
+import styled from 'styled-components';
+import {
+  Button,
+  Link,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+  FormLabel,
+  RadioGroup,
+  Radio,
+  Box as MuiBox,
+  Select,
+  InputLabel,
+} from '@material-ui/core';
+import { File } from '../../../types/license.types';
+import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { useTranslation } from 'react-i18next';
+
+const StyledTitle = styled.div`
+  font-weight: bold;
+`;
+const StyledDescription = styled.div`
+  font-style: italic;
+`;
+const StyledActions = styled.div``;
+
+const StyledFormControl = styled(FormControl)`
+  width: 30%;
+  margin-top: 1rem;
+`;
+
+const StyledLicenseFormControl = styled(StyledFormControl)`
+  margin-top: 2rem;
+`;
```

Comment:
Må gjøre dette siden vi har margin-top satt for alle TextFields i theme. I dette tilfellet er det en TextField og en Select som ikke kommer på linje by default siden TextField starter med margin
Jeg stemmer for at vi fjerner det fra theme, så slipper vi å måtte gjøre sånne rare ekstrasteg her og der

Reply:
enig. vi kan heller wrappe de tekstfeltene som trenger lik padding med en felles komponent

Reply:
enig!